src/Security/Security/Models/SqlVulnerabilityAssessment/PSSqlVulnerabilityBenchmarkReference.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.SecurityCenter.Models.SqlVulnerabilityAssessment
+{
+    public class PSSqlVulnerabilityBenchmarkReference
+    {
+        public string Benchmark { get; set; }
+        public string Reference { get; set; }
+
+        public override string ToString()
+        {
+            return $"{{\r\n" +
```

Comment:
Please replace \r\n with Environment.NewLine to be compatible in different platforms.

Reply:
pending